docs/access_rights/organization_account.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Organization account
+description: Create a specialized account for your team to encourage collaboration and manage permissions efficiently. Convert an existing account or create one from scratch.
+menuWeight: 8.1
+paths:
+    - access-rights/organization-account
+---
+
+# Organization account
+
+Apify's organization account allows you to keep your business account separate from your personal account. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
```

Comment:
It mainly allows collaboration of multiple team members, I think that this should be the main point